charter/opencdms_charter.md

```diff
@@ -92,7 +92,9 @@ As an open source project, OpenCDMS will be delivered by a range of contributors
 
 OpenCDMS will encourage contributions from Members of WMO and of the Association of Hydro-Meteorological Equipment Industry (HMEI) as well as non-profit organizations, universities, the Food and Agricultural Organization, and other interested bodies and individuals. 
 
-Users in the NMHSs will be encouraged to participate as Contributors, especially from their experience, and in the design, testing and usage discussions. There will be an emphasis on inclusion and mentoring for contributors based in developing and least-developed countries while being vigilant on the use of the best adapted and most appropriate technologies.
+Users in the NMHSs but also from any other organisation managing environmental data will be encouraged to participate as Contributors, especially from their experience, and in the design, testing and usage discussions. ["There will be an emphasis on inclusion and mentoring for contributors based in developing and least-developed countries while being vigilant on the use of the best adapted and most appropriate technologies". Here also I think it is not needed to write this in the Project Charter. The objective is to get an efficient system taking account the worldwide experience of all. it seems necessary to work with all users and developers enthusiastic about this project, whether they come from Africa or from Silicon Valley. I also think that the latest information technologies require pure professional developers. It is no longer up to meteorologists to be computer code developers as well. This work has become far too specific. I do not say that contrinutors should not come from developing and least developing countries, on the contrary. I say that the desire to participate to he project, the expertise, the competence, the meritocracy are the basement of a sustanaible project.
```

Comment:
"other interested bodies" are already covered in the previous paragaph. Given that the developing and least developed countries are where the problems are greatest, the project should make this "an emphasis" but not to the exclusion of other needs. I would rather keep the original wording.